CHANGELOG.md

```diff
@@ -1,49 +1,70 @@
 # Changelog
+
```

Comment:
The linter was going nuts here, so I let it fix some of the changelog syntax. Half of the code is just this changelog

Reply:
:thinking: There is no linter in this repo afaik. Were you running rubocop generally?

Not a bad thing, it does look more consistent...